config/locales/account/en.yml

```diff
@@ -73,6 +73,7 @@ en:
     login:
       ie_not_supported: Internet Explorer 11 is no longer supported as of %{date}
       piv_cac: Sign in with your government employee ID
+      tab_navigation: Account tab navigation
```

Comment:
Still waiting on confirmation / translations for this label, so there'll be a spec failure 'til that's settled.

Reply:
Added confirmed label in ed847ed